exo-go/src/service_config_helpers/index.go

```diff
@@ -0,0 +1,26 @@
+package serviceConfigHelpers
+
+import (
+	"io/ioutil"
+	"log"
+	"path"
+
+	yaml "gopkg.in/yaml.v2"
+
+	"github.com/Originate/exosphere/exo-go/src/types"
+)
+
+// GetServiceConfig reads service.yml of the role service and returns
+// the serviceConfig object
+func GetServiceConfig(location string) types.ServiceConfig {
+	yamlFile, err := ioutil.ReadFile(path.Join(location, "service.yml"))
+	if err != nil {
+		log.Fatalf("Failed to read service.yml: %s", err)
```

Comment:
Please return an error instead of using log.Fatal. Same below. 

I also don't think its worth wrapping this error. Since we pass in a file name that includes `service.yml` I expect the error message would give us enough detail. For unmarshalling the yaml, I think the error is useful since that error message would not give us the context about what is being unmarshalled.